packages/loopring_v3/contracts/impl/ExchangeV3.sol

```diff
@@ -654,49 +507,39 @@ contract ExchangeV3 is IExchangeV3
         view
         returns (bool)
     {
-        return owner == agent || state.agent[owner][agent];
+        return owner == agent || state.agent[owner][agent] || state.whitelistedAgent[agent];
```

Comment:
switch the last checks, as `state.whitelistedAgent[agent]` might be more frequently used.

Reply:
Seems like solidity does indeed support short circuiting for booleans!

Reply:
Yes, what I'm saying is that the odds of `state.whitelistedAgent[agent]==true` is bigger than `state.agent[owner][agent] == true`. So switching the order of these two will save some gas.

Reply:
Yes, already changed!